.env

```diff
@@ -106,13 +108,15 @@ NEXT_PUBLIC_L1_PUFFER_GATEWAY_PROXY_ADDR = ""
 NEXT_PUBLIC_L2_PUFFER_GATEWAY_PROXY_ADDR = ""
 NEXT_PUBLIC_L1_STANDARD_ERC20_GATEWAY_PROXY_ADDR = ""
 NEXT_PUBLIC_L2_STANDARD_ERC20_GATEWAY_PROXY_ADDR = ""
-NEXT_PUBLIC_L1_SCROLL_MESSENGER = "0x5EF9a083704451F60d76d1aCf59b7d9E4Ae8CAC8"
-NEXT_PUBLIC_L2_SCROLL_MESSENGER = "0x703848F4c85f18e3acd8196c8eC91eb0b7Bd0797"
-NEXT_PUBLIC_L1_GATEWAY_ROUTER_PROXY_ADDR = "0x1355929967cff13127E168C7d34bdddd8e4bfefB"
-NEXT_PUBLIC_L2_GATEWAY_ROUTER_PROXY_ADDR = "0x373E0B8B80A15cdf587C1263654c6B5edd195a43"
-NEXT_PUBLIC_SCROLL_CHAIN = "0x6025bBC40539619c739779A8c43579E8F1CD923d"
-NEXT_PUBLIC_L1_MESSAGE_QUEUE = "0x91891f62174C61ebA42Fa931C80363ef326Ae6B0"
-NEXT_PUBLIC_L1_MESSAGE_QUEUE_WITH_GAS_PRICE_ORACLE="0x91891f62174C61ebA42Fa931C80363ef326Ae6B0"
+NEXT_PUBLIC_L1_SCROLL_MESSENGER = "0xBf3c160CD7Da23d5fbE169b15001d5BD89b8Fd33"
+NEXT_PUBLIC_L2_SCROLL_MESSENGER = "0x330C398469073471fEC999D722F06954aD402f31"
+NEXT_PUBLIC_L1_GATEWAY_ROUTER_PROXY_ADDR = "0x105CC0295D2D7f403f1A08780E01528cB3137144"
+NEXT_PUBLIC_L2_GATEWAY_ROUTER_PROXY_ADDR = "0xA94C35dF72f4e6A2E19C0f5616D82cAdf5B628D2"
+NEXT_PUBLIC_SCROLL_CHAIN = "0x59315E81459167C96A544f21ec0cbB2dEa7feb73"
+NEXT_PUBLIC_L1_MESSAGE_QUEUE = "0x1765af3fc36A8b9E4cD57066bBde2Cff2cb2d14f"
+NEXT_PUBLIC_L1_MESSAGE_QUEUE_WITH_GAS_PRICE_ORACLE="0x1765af3fc36A8b9E4cD57066bBde2Cff2cb2d14f"
+NEXT_PUBLIC_L2_ERC20_WETH_ADDR = "0x09ffc24E27Ec08D98032e843b28EA9aF43e21525"
```

Comment:
This shouldn't be hardcoded. There are two methods in the ERC20Gateways to lookup the addresses onchain:
https://github.com/t1protocol/t1/blob/cbae7537e9d397c7891a035e807f613aa73fc591/contracts/src/L2/gateways/IL2ERC20Gateway.sol#L42
https://github.com/t1protocol/t1/blob/cbae7537e9d397c7891a035e807f613aa73fc591/contracts/src/L1/gateways/IL1ERC20Gateway.sol#L48

Perhaps we can have a list of supported tokens and get addresses onchain.